espnet2/uasr/generator/conv_generator.py

```diff
@@ -67,6 +67,7 @@ def __init__(
             self.pad = cfg.generator_pad
             self.bias = cfg.generator_bias
             self.dropout = torch.nn.Dropout(cfg.generator_dropout)
+            # TODO: fix batch_norm
```

Comment:
Could you make it the format `TODO(your_name): comment` so that we can locate the issue with POC

Reply:
done